base/essentials.jl

```diff
@@ -1250,6 +1250,50 @@ function isiterable(T)::Bool
     return hasmethod(iterate, Tuple{T})
 end
 
+"""
+    @world(sym, world)
```

Comment:
Should we attempt to merge this with `Base.@invokelatest world M.a`?

Reply:
@IanButterworth suggested that above. The syntax is reserved, but I'm not convinced.

Reply:
I am not convinced either, just observing the symmetry that the two different macros return the same result, particularly if we allow `@world` to handle call syntax or `@invokelatest` to handle bindings